lib/miq_expression.rb

```diff
@@ -344,6 +344,15 @@ def sql_supports_atom?(exp)
         # TODO: Support includes operator for sub-sub-tables
         return false
       end
+    when "includes any", "includes all", "includes only"
```

Comment:
Are we planning on implementing includes all and includes only in the future?

Reply:
So my thought with this is that they would all fall in the same vain eventually, so just put them here for now since this is probably where those three would go in the future anyway.  Yes, it is extra lines of code now, but in the future it would be how we would probably be where we through this anyway, so figured I would just save the effort of remembering to do that later.

That said, I could look into `includes all` and `includes only` implementations if you think that has a use.

Reply:
I suppose it doesn't matter as long as we have tests demonstrating that each goes through `arel` or ruby based upon the operator.